pkg/license/license.go

```diff
@@ -73,6 +73,7 @@ func (r LicensingResolver) ToInfo(totalMemory resource.Quantity) (LicensingInfo,
 }
 
 // Save updates or creates licensing information in a config map
+// This relies on UnconditionalUpdates being supported configmaps and may change in k8s v2: https://github.com/kubernetes/kubernetes/issues/21330
```

Comment:
This is just a note for the future since it is currently failing on the new version of controller-runtime (since we are not passing an RV in the update call)

Reply:
Should we just fix the function to do a get + create or update instead? (maybe in a follow-up issue)

Reply:
I wondered about that, but we really don't care about the current state when updating the configmap so I'm not sure that would be a useful change (except that it would let us use the current fake client and update controller-runtime)